### PR TITLE
db provision script should insert all the products (#81)

### PR DESCRIPTION
**Pull request by damdiana**

### CODE REVIEW

File: db_provision.mjs [hunk 1]
```diff
@@ -4,13 +4,19 @@
 import { db } from "@vercel/postgres";
 import * as dotenv from "dotenv";
 dotenv.config({
-  path: ".env.development.local",
+  path: ".env.local",
```
Comment (iampava) [inline db_provision.mjs:7]: Why this change here? Did you rename the local environments file from `.env.development.local` to `.env.local`?

File: db_provision.mjs [hunk 2]
```diff
@@ -4,13 +4,19 @@
...
 });
 
 const client = await db.connect();
 
 // createProductsTable();
-fetchAndInsertProducts("mask");
+await fetchAndInsertProducts("mask");
```
Comment (iampava) [inline db_provision.mjs:13]: I suggest you merge this PR before https://github.com/damdiana/beauty-app/pull/84. Both target the same file, and you will have conflicts. But I think it's easier to merge this one first, and then the other one.

**Review by iampava — CHANGES_REQUESTED** [review]

File: db_provision.mjs (hunk 1 above)
Comment (damdiana) [reply db_provision.mjs:7]: Yes. The file name is now .env.local

Comment (damdiana) [reply db_provision.mjs:7]: changed the name back to .env.development.local
